osu.Game/OsuGameBase.cs

```diff
@@ -187,8 +187,9 @@ private void load(ReadableKeyCombinationProvider keyCombinationProvider)
             Resources.AddStore(new DllResourceStore(OsuResources.ResourceAssembly));
 
             dependencies.Cache(contextFactory = new DatabaseContextFactory(Storage));
+            dependencies.Cache(RulesetStore = new RulesetStore(contextFactory, Storage));
```

Comment:
After this move, `RulesetStore` now attempts to access DB tables before `runMigrations()` is ran on line 207 in this BDL, causing the rather spectacular CI failure.

Reply:
I figured something like this would be the case 😅